sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -546,6 +568,16 @@ def _close_handler(self):
         self._message_iter = None
         super(ServiceBusReceiver, self)._close_handler()
 
+    def _keep_receiving(self):
+        while not self._shutdown.is_set() and self._handler:
+            try:
+                self._handler._connection.work()  # pylint: disable=protected-access
+                time.sleep(5)
+                _LOGGER.debug("Keep receiving alive")
```

Comment:
Maybe this should be something like "Keep connection alive"

Reply:
yep, that sounds good to me